CMake/ViskoresConfig.cmake.in

```diff
@@ -180,6 +171,14 @@ if(Viskores_ENABLE_CUDA)
   endif()
 endif()
 
+if (TARGET viskores::cuda OR TARGET viskores::kokkos_cuda)
+  enable_language(CUDA)
+endif()
+
+if (TARGET viskores::kokkos_hip)
+  enable_language(HIP)
+endif()
+
```

Comment:
We cannot enable language in the cmake configs since we:

```
enable_language* must be called in the highest directory common to all targets using the named language directly for compiling sources or indirectly through link dependencies. It is simplest to enable all needed languages in the top-level directory of a project.
```

https://cmake.org/cmake/help/latest/command/enable_language.html

Reply:
Good point @vicentebolea. In that case, I am going to abandon this PR. It looks like the languages get enabled when `ViskoresWrappers.cmake` is included, and you need to include that anyway if you have device code. So, maybe this is not a necessary change.